### DISCUSSION TITLE
GitHub Pages

### DISCUSSION THREAD

**Discussion by ShitijHalder** [community/community#49219] [Other Feature Feedback, Questions, & Ideas] [answered]
Okay so 1 year ago I created a website - [shitijhalder.github.io/shitij-in-nutshell]() .

This website was about me and personal information, The last day I deleted the repo and cleared everything and deleted the page too.

The same day I cloned the website and redesigned it before deleting it and I made a new repo, this time I set the repo name as # # `shitijhalder.github.io` and GitHub automatically set the new page to [shitijhalder.github.io]() , now I didn't setup anything but It created the site automatically, I didn't think much about that I finished the website and released the files there.

Now everything is okay and fine. My new page is ready to be seen and everything is working well.

Now I was going through some old files and I accidentally clicked on my old pages link [shitijhalder.github.io/shitij-in-nutshell]() , I was surprised to see that this website is still running but without any single layout and CSS and anything, there was just a simple running HTML. 

Now it's just so frustrating, is there any way to remove the old page? I deleted it but it's still on.

Help!!

![Screenshot_20230305-142701_Chrome.png](https://user-images.githubusercontent.com/89300443/222951233-4fbbe8cf-2846-42d1-a3f0-542de37f9d79.png)

This is the old link and it's still up.

New one is this below 👇

![Screenshot_20230305-142756_Chrome.png](https://user-images.githubusercontent.com/89300443/222951280-2be58866-fc80-43bd-b43b-73d286d1a08c.png)

**Comment by jge162** [comment]
Sounds like you deleted the old one but did not disable it. You need to go back to the settings (source) and disable the old one.

**Reply by ShitijHalder** [reply]
I couldn't find the settings of that thing, if you know where I can find the settings then please reply with steps.

**Reply by jge162** [reply]
Reach out to support. They can help you sort it out.

**Reply by ShitijHalder** [reply]
> Reach out to support. They can help you sort it out. 

Okay, thanks for the information!

**Comment by Neodevils** [comment, marked as answer]
You have to disable GitHub Pages on your "**shitij-in-nutshell**" repository. It'll fix your issue.

*If this helps, don't forget to mark the discussion as solved!* 🙌🏻

**Reply by ShitijHalder** [reply]
Actually the shitij-in-nutshell repository isn't even available in my account or anywhere, so how do I disable the GitHub page?

**Reply by Neodevils** [reply]
> Actually the shitij-in-nutshell repository isn't even available in my account or anywhere, so how do I disable the GitHub page?

Hmm, maybe try to restore your repository and disable GitHub Pages on it.

**Reply by ShitijHalder** [reply]
> > Actually the shitij-in-nutshell repository isn't even available in my account or anywhere, so how do I disable the GitHub page?
> 
> Hmm, maybe try to restore your repository and disable GitHub Pages on it. 

I understand but at first I deleted the repository so how would I retrieve it? It's deleted... And if there is any option for that then please let me know about it.

**Reply by jge162** [reply]
As I mentioned above. Support can help you with a deleted repository and can help fix issue with current pages. Reach out to them.

**Reply by ShitijHalder** [reply]
> As I mentioned above. Support can help you with a deleted repository and can help fix issue with current pages. Reach out to them. 

Okay.

**Reply by Neodevils** [reply]
> > > Actually the shitij-in-nutshell repository isn't even available in my account or anywhere, so how do I disable the GitHub page?
> > 
> > 
> > Hmm, maybe try to restore your repository and disable GitHub Pages on it.
> 
> I understand but at first I deleted the repository so how would I retrieve it? It's deleted... And if there is any option for that then please let me know about it.

Go to your settings, then go to repositories. You'll be seeing deleted repositories.

**Reply by ShitijHalder** [reply]
> > > > Actually the shitij-in-nutshell repository isn't even available in my account or anywhere, so how do I disable the GitHub page?
> > > 
> > > 
> > > Hmm, maybe try to restore your repository and disable GitHub Pages on it.
> > 
> > I understand but at first I deleted the repository so how would I retrieve it? It's deleted... And if there is any option for that then please let me know about it.
> 
> Go to your settings, then go to repositories. You'll be seeing deleted repositories.

Thanks for your help!! The repository is back now... Now I can finally delete the previous GitHub Pages. Once again thank you for your time 😃

**Reply by Neodevils** [reply]
> > > > > Actually the shitij-in-nutshell repository isn't even available in my account or anywhere, so how do I disable the GitHub page?
> > > > 
> > > > 
> > > > Hmm, maybe try to restore your repository and disable GitHub Pages on it.
> > > 
> > > 
> > > I understand but at first I deleted the repository so how would I retrieve it? It's deleted... And if there is any option for that then please let me know about it.
> > 
> > 
> > Go to your settings, then go to repositories. You'll be seeing deleted repositories.
> 
> Thanks for your help!! The repository is back now... Now I can finally delete the previous GitHub Pages. Once again thank you for your time 😃

You're welcome! 🤗 Since your issue seems solved, don't forget to mark this discussion as solved. 🤠
